amd_openvx/openvx/ago/ago_internal.h

```diff
@@ -609,7 +609,7 @@ struct AgoNode {
     struct { bool enable; int paramIndexScalar; int paramIndexArray; } gpu_scalar_array_output_sync;
 #if ENABLE_OPENCL
     vx_uint32 opencl_type;
-    char opencl_name[VX_MAX_KERNEL_NAME];
+    char opencl_name[VX_MAX_KERNEL_NAME+8];
```

Comment:
Instead of adding +8 to VX_MAX_KERNEL_NAME here, can you increase the VX_MAX_KERNEL_NAME itself?

Reply:
This variable comes from the API Doc, we can not modify it.